packages/react/src/utils/generate-component-classes.ts

```diff
@@ -0,0 +1,12 @@
+const DS_CLASS_PREFIX = 'eds-';
```

Comment:
J'ai mis `eds` pour Equisoft Design System mais je suis ouvert aux propositions🤷‍♂️

Reply:
Un jour faudrait qu'on se trouve un p'tit nom plutôt que "DS"...

Reply:
> Un jour faudrait qu'on se trouve un p'tit nom plutôt que "DS"...

soon to come